app/views/layouts/_footer.html.erb

```diff
@@ -2,11 +2,8 @@
   <div class="footer-top">
     <div class="container">
       <div class="row">
-        <div class="col-lg-4 col-md-12">
-          <p align=center>
-          <%= image_tag("logo.png", alt: "NUANSA Cultural Productions", size: "300x100") %>
-          <br>
-          </p>
+        <div class="col-lg-4 col-md-12 footer-info">
+          <%= image_tag("logo.png", alt: "NUANSA Cultural Productions") %>
```

Comment:
But now the logo is not at the centre for mobile version (?)
<img width="357" alt="Screen Shot 2019-06-02 at 13 25 44" src="https://user-images.githubusercontent.com/28918289/58757187-4b557580-853a-11e9-968b-2c9082c7baca.png">